tests/network/services.go

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
Right now, the files on `kubevirt/tests/network/` package are:
  - framework
  - primary_pod_network (afaiu, you can only have a single *primary* pod network).

So I think there are things that make sense to be singular, whereas others make sense to be plural - just like language itself. E.g. "migration tests" ; "services tests"; "primary pod network tests"; "binding mechanisms tests"; ...

Reply:
No. I suggest to use meaning words, whose cardinality will differ depending on the subject at hand.

What do you gain by enforcing all files here will be singular or plural ? Honestly interested.

Reply:
i also don't think we should enforce, but use the native english for each case

Reply:
> What do you gain by enforcing all files here will be singular or plural ? Honestly interested.

Consistency.

> i also don't think we should enforce, but use the native english for each case

How is this related to native English exactly? One either uses single naming or plural naming to describe something that is a collection.

If it was a class or a package, it would have matter more. You can say "services.pool" or "service.pool", etc.

Reply:
> > What do you gain by enforcing all files here will be singular or plural ? Honestly interested.
> 
> Consistency.

Consistency break the freedom to give the right name ad hoc. 
This is also the answer for the question below.
I think we should strive to give the right name ad hoc, and not enforce plurality.
 

> 
> > i also don't think we should enforce, but use the native english for each case
> 
> How is this related to native English exactly? One either uses single naming or plural naming to describe something that is a collection.
> 
> If it was a class or a package, it would have matter more. You can say "services.pool" or "service.pool", etc.